README.md

```diff
@@ -38,7 +39,7 @@ service/environment.
 var config = new ZipkinConfig
 {
 	Bypass = request => request.Uri.AbsolutePath.StartsWith("/allowed"),
-	Domain = new Uri("https://yourservice.com"),
+	Domain = request => new Uri("https://yourservice.com"), // or, you might like to derive a value from the request, like r => new Uri($"{r.Scheme}://{r.Host}"),
```

Comment:
@petemounce Sorry for this late reply. Adhoc this on local, and proves ok. Thanks!
Just one request, can you change the alternative example to --> 
new Uri($"{r.Uri.Scheme}{Uri.SchemeDelimiter}{r.Uri.Host}") ?